app/services/invoices/refresh_draft_and_finalize_service.rb

```diff
@@ -68,5 +70,9 @@ def should_deliver_email?
       License.premium? &&
         invoice.organization.email_settings.include?('invoice.finalized')
     end
+
+    def flag_lifetime_usage
+      LifetimeUsages::FlagRefreshFromInvoiceService.call(invoice:).raise_if_error!
```

Comment:
Looking into it. I had to use plural form for module to avoid conflict with model class

Reply:
Same, I think we can just live with that.